labs/sysdev-tinysystem/sysdev-tinysystem.tex

```diff
@@ -113,15 +113,24 @@ \section{Booting the system}
 filesystem over NFS:
 
 \if\defstring{\labboard}{beagleplay}
-\begin{verbatim}
-VFS: Mounted root (nfs filesystem) on device 0:20.
-\end{verbatim}
+  \begin{verbatim}
+  VFS: Mounted root (nfs filesystem) on device 0:20.
+  \end{verbatim}
 \else
-\begin{verbatim}
-VFS: Mounted root (nfs filesystem) on device 0:16.
-\end{verbatim}
+  \if\defstring{\labboard}{stm32mp2}
+    \begin{verbatim}
+    VFS: Mounted root (nfs filesystem) on device 0:22.
+    \end{verbatim}
+  \else
+    \begin{verbatim}
+    VFS: Mounted root (nfs filesystem) on device 0:16.
+    \end{verbatim}
+  \fi
```

Comment:
This is a lot of noise for nothing. Just replace it (in a separate commit) with something that isn't board specific by using "on device X:Y" or "on device [...]" or something like that.